lib/game_logic.rb

```diff
@@ -0,0 +1,44 @@
+# rubocop: disable Style/Documentation, Style/GuardClause, Layout/LineLength, Metrics/MethodLength
+# frozen_string_literal: true
+
+module GameLogic
+  def display_board(board)
+    puts " #{board[0]} | #{board[1]} | #{board[2]} "
+    puts '-----------'
+    puts " #{board[3]} | #{board[4]} | #{board[5]} "
+    puts '-----------'
+    puts " #{board[6]} | #{board[7]} | #{board[8]} "
+  end
+
+  def valid_input(input, arr)
+    if (input.nil? || input.is_a?(Integer) == false || (1..9).include?(input) == false) || (arr[input - 1] == 'X' || arr[input - 1] == 'O')
```

Comment:
This ```input.is_a?(Integer) == false``` is equal to ```!input.is_a?(Integer)``` the only addition is the ```!``` exclamation sign. this way you can remove the false to make the line shorter.